### DISCUSSION TITLE
Cant verify anything

### DISCUSSION THREAD

**Discussion by Alvaroinfantee** [community/community#54898] [GitHub Education]
I have tried everything. In my last attempt I uploaded a certificate of enrollment with an English translation on top of it. Everything with my name, the date, the name of the institution and the signature of the registrar and still wont even get past the bot. Is there anything I can do?

**Comment by iamreiyn** [comment]
Hi @Alvaroinfantee 

Could you tell us about the exact error that you're facing? 

Ensure that your evidence matches the information on your GitHub profile. The verification process will check your GitHub username, email, name, and other details to confirm the evidence submitted. To avoid differences, use the same name or nickname in your application materials. If you update your profile information, log out and log back in before re-applying.

And I can see that you don't have any bio added to your profile, consider adding it as it is required as well.

Let us know how your application goes!

**Reply by Alvaroinfantee** [reply]
Hello
I have just tried it. Still no luck. I checked my profile and everything matches. I logged out and then logged back in. Still get the same error. The error message is "your application cannot be reviewed until you fix the following:
Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy.
Your image should include one of the following:
Dated school ID"

I find this weird since I checked the "dated enrollment letter" on the drop-down.

**Reply by iamreiyn** [reply]
Try to take a picture of the translated document alongside the original.
Otherwise, I would recommend trying with another means to verify your academic status.

> When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.
> 
> If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

[Source](https://github.com/orgs/community/discussions/54169)

**Reply by Alvaroinfantee** [reply]
Thanks

**Comment by samus-aran** [comment]
Hi @Alvaroinfantee 👋 

We're happy you're here!

Looking at your latest rejection reasons we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
